crates/core/component/distributions/src/component/view.rs

```diff
@@ -21,8 +21,15 @@ pub trait StateReadExt: StateRead {
     }
 
     // Get the total amount of LQT rewards issued for this epoch.
-    fn get_lqt_reward_issuance_for_epoch(&self) -> Option<Amount> {
-        self.object_get(&state_key::lqt_reward_issuance_for_epoch())
+    async fn get_lqt_reward_issuance_for_epoch(&self, epoch_index: u64) -> Option<Amount> {
+        let key = &state_key::lqt_reward_issuance_for_epoch(epoch_index);
+
+        self.nonverifiable_get(&key.as_bytes())
+            .await
+            .expect(&format!(
```

Comment:
FWIW: the usage is `expect(INVARIANT)` where `INVARIANT` is the reason why it is okay to panic.